uiplib/scrape.py

```diff
@@ -28,14 +28,17 @@ def dlProgress(count, blockSize, totalSize):
     sys.stdout.write("\r" + "...%d%%" % percent)
     sys.stdout.flush()
 
-
-def get_images(url, directory):
+def get_images(url, directory, count):
     '''
     scrape images from /r/wallpapers
     '''
     image_links = []
     p_url = url
-    while len(image_links) < NUMBER_OF_IMAGES_TO_PARSE:
+
+    # this specifies the number of images to be parsed
+    no_of_images = NUMBER_OF_IMAGES_TO_PARSE if not count else int(count)
```

Comment:
hey can u make this change in setup.py to add NUMBER_OF_IMAGES_TO_PARSE in the `settings.json` so we can remove this from `settings.py` and have a default value for that if no one specified this flag?